app/Models/Foundation/UserStories/UserStory.php

```diff
@@ -0,0 +1,341 @@
+<?php namespace App\Models\Foundation\UserStories;
+/*
+ * Copyright 2024 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping AS ORM;
+use models\main\File;
+use models\main\Organization;
+use models\main\Tag;
+use models\summit\SummitAbstractLocation;
+use models\utils\One2ManyPropertyTrait;
+use models\utils\SilverstripeBaseModel;
+
+/**
+ * @ORM\Entity(repositoryClass="repositories\main\DoctrineUserStoryRepository")
+ * @ORM\Table(name="UserStoryDO")
+ * Class File
+ * @package App\Models\Foundation\UserStories
+ */
+class UserStory extends SilverstripeBaseModel
+{
+    use One2ManyPropertyTrait;
+
+    protected $getIdMappings = [
+        'getOrganizationId' => 'organization',
+        'getIndustryId'     => 'industry',
+        'getLocationId'     => 'location',
+    ];
+
+    protected $hasPropertyMappings = [
+        'hasOrganization' => 'organization',
+        'hasIndustry'     => 'industry',
+        'hasLocation'     => 'location',
+    ];
+
+    /**
+     * @ORM\Column(name="Name", type="string")
+     */
+    private $name;
+    /**
+     * @ORM\Column(name="Description", type="string")
+     */
+    private $description;
+    /**
+     * @ORM\Column(name="ShortDescription", type="string")
+     */
+    private $short_description;
+    /**
+     * @ORM\Column(name="Link", type="string")
+     */
+    private $link;
+
+    /**
+     * @ORM\Column(name="Active", type="boolean")
+     * @var bool
+     */
+    private $is_active;
+
+    /**
+     * @ORM\Column(name="MillionCoreClub", type="boolean")
+     * @var bool
+     */
+    private $is_million_core_club;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="models\main\Organization")
+     * @ORM\JoinColumn(name="OrganizationID", referencedColumnName="ID")
+     * @var Organization
+     */
+    private $organization;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Models\Foundation\UserStories\UserStoriesIndustry", inversedBy="user_stories")
+     * @ORM\JoinColumn(name="IndustryID", referencedColumnName="ID")
+     * @var UserStoriesIndustry
+     */
+    protected $industry;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="models\summit\SummitAbstractLocation")
+     * @ORM\JoinColumn(name="LocationID", referencedColumnName="ID")
+     * @var SummitAbstractLocation
+     */
+    protected $location;
```

Comment:
this is not correct
its continent please do review the mapping
https://github.com/OpenStackweb/openstack-org/blob/ebca0d466b5bf217107563f8cacc217ec58d4f75/user-stories/code/infrastructure/active_records/UserStoryDO.php#L30